### ISSUE TITLE
Timeline of Bible History compilation

### ISSUE THREAD

**Issue by erykjj** [erykjj/jwpublib#29]
I am looking for volunteers to add scriptural references to events that haven't got them yet, and others to verify these refences.

**Comment by chessurisme** [comment]
```md
Enosh born - Genesis 5:6
Jared born - Genesis 5:15
Enosh dies - Genesis 5:11
Jared dies - Genesis 5:20
Lamech dies - Genesis 5:31
Downpour stops - Genesis 8:2
Ark comes to rest in Ararat - Genesis 8:4
Mountaintops visible - Genesis 8:5
Noah removes part of roof - Genesis 8:13
Exit from ark - Genesis 8:15-19
Sarah born - Genesis 17:17
```

**Comment by erykjj** [comment]
Whoa!! :-)
Slow down.
I'll post a list once I get some volunteers :-)

**Comment by chessurisme** [comment]
I invited the JWTalk members for this.

**Comment by erykjj** [comment]
Here are the sets of events:
[events (1).csv](https://github.com/erykjj/jwpublib/files/13943668/events.1.csv)
[events (2).csv](https://github.com/erykjj/jwpublib/files/13943669/events.2.csv)
[events (3).csv](https://github.com/erykjj/jwpublib/files/13943670/events.3.csv)
[events (4).csv](https://github.com/erykjj/jwpublib/files/13943671/events.4.csv)

If you desire to participate, please indicate in a comment which set you take on.

Just add the scripture reference after the tab at the end of each line. Keep it as specific as possible: the precise verse(s) instead of a whole account.

If there are no biblical references to the event, put N/A or something to that effect.

And, of course, reupload in this thread as soon as you finish.

Thank you in advance!

**Comment by chessurisme** [comment]
The csv looks like this in MS Excel 

![image](https://github.com/erykjj/jwpublib/assets/137697011/dabaf25e-468d-41a8-8cb3-b7867649d7ae)

**Comment by erykjj** [comment]
You have to select to split on **tabs** when you open it

**Comment by chessurisme** [comment]
When I open it, there's no option to select to split on tabs, I don't know why.

**Comment by erykjj** [comment]
Try **right-clicking the file to open with a text editor** - easiest.
Or, open Excel directly, then File > Open... and select
![2024-01-15_20-21-46](https://github.com/erykjj/jwpublib/assets/107011362/4970b944-c1c3-4242-a4ef-014ef7debcf9)
That might work.

**Comment by erykjj** [comment]
I'll re-upload those files as excel files. Give me a moment.

**Comment by erykjj** [comment]
Excel files [above](https://github.com/erykjj/jwpublib/issues/29#issuecomment-1892867154).

**Comment by erykjj** [comment]
Cleaned up the thread a bit as the deleted comments don't apply to the Excel format now posted above.

**Comment by Frankygishere** [comment]
[BibleTimelinePlus.pdf](https://github.com/erykjj/jwpublib/files/13999805/BibleTimelinePlus.pdf)
Not sure if this could be of any help?

**Comment by erykjj** [comment]
> [BibleTimelinePlus.pdf](https://github.com/erykjj/jwpublib/files/13999805/BibleTimelinePlus.pdf) Not sure if this could be of any help?

Thank you, @Frankygishere 

It's nice to see others taking an interest in Bible chronology :-)
It seems to be fairly reliable information (though I did find a couple of errors).

I guess I should clarify that it's not that I "_need help_" as such - I am simply giving other the opportunity to contribute to the project. Eventually, I will complete the missing references as part of my own research (and the above-mentioned source may ba a good starting point).

**Comment by jeremy-j-t** [comment]
@erykjj
Hi! I'd like to help. Which sheet should I start with?

**Comment by erykjj** [comment]
Hi @jeremy-j-t ,

Thank you! You can pick any. No one has "officially" claimed anything yet. Just put in the comments here which one you are taking on.

**Comment by jeremy-j-t** [comment]
Great!
events.2.xlsx looks good
I'll be back in a few days 😀

**Comment by chessurisme** [comment]
Oh, also I am in events1.xlsx just please be patient to me. I forgot to tell you, Eryk, about this.

**Comment by erykjj** [comment]
> Oh, also I am in events1.xlsx just please be patient to me. I forgot to tell you, Eryk, about this.

No rush. Thank you!

**Comment by erykjj** [comment]
Hi @jeremy-j-t @chessurisme 

How are you managing? Need a hand?

**Comment by chessurisme** [comment]
Oh really sorry for late reply, I have been replying this and when I see it again, my reply is gone... I am not sure why... But yeah about the stuff, I will send it to you this weekdays, I am out in ministry, so can't do this at the moment... Thanks for patience, dearly Eryk.

**Comment by chessurisme** [comment]
[events.1.xlsx](https://github.com/erykjj/jwpublib/files/14195453/events.1.xlsx)
Done!

**Comment by erykjj** [comment]
> [events.1.xlsx](https://github.com/erykjj/jwpublib/files/14195453/events.1.xlsx) Done!

Excellent, my brother! Just had a quick look and... VERY GOOD :-)

**Comment by jeremy-j-t** [comment]
Hey guys! Still at it...so much for a few days, lol. Probably about a third deep.

**Comment by erykjj** [comment]
Hi @jeremy-j-t 
Please don't stress too much over this :-)

Why don't you send me what you have done so far. Perhaps I'll get you to double-check some references for me instead. It's quicker as you only have to look up the scripture (and perhaps reduce it just to the key verse(s)).

**Comment by erykjj** [comment]
@chessurisme I just release v4 which includes your contributions ;-)

@jeremy-j-t I haven't forgotten you. I will add your results in a future update, so take your time :-)

**Comment by chessurisme** [comment]
> @chessurisme I just release v4 which includes your contributions ;-)

Quite interesting!

**Comment by chessurisme** [comment]
Notifying the JWTalk Community! 😗

**Comment by erykjj** [comment]
> Hey guys! Still at it...so much for a few days, lol. Probably about a third deep.

Hey @jeremy-j-t ,

Making any headway? Why don't you send me what you have and we'll figure out how to do the rest ;-)

**Comment by jeremy-j-t** [comment]
Hey bro, good idea.

Sorry, I've been on the move for the last week and a half, and out of signal the last few days. Just had a look at what I had left and I realised I was basically done, most of the later events aren't mentioned.

Here's what I've got. Notice the highlighted rows...they are interesting.
Them and some others...I guess there isn't exactly a verse referring to the event, but the verse either provides some degree of time context or is prophetic, and I suppose would be of value in the timeline. See what you reckon.

[events.2.xlsx](https://github.com/erykjj/jwpublib/files/14403958/events.2.xlsx)

**Comment by jeremy-j-t** [comment]
@erykjj ...I'm github rookie... is there a way to send dm?

**Comment by chessurisme** [comment]
Looks good to me.

**Comment by erykjj** [comment]
[events.xlsx](https://github.com/erykjj/jwpublib/files/14405794/events.xlsx)

Just missing these few... Can one of you have a go?

**Comment by chessurisme** [comment]
I'll have a go!

**Comment by chessurisme** [comment]
[events.xlsx](https://github.com/erykjj/jwpublib/files/14413158/events.xlsx)

**Comment by erykjj** [comment]
Hi Kyle,
Thank you, but this will take a bit more time, I'm afraid... Just a quick look, you are giving a reference for Darius the Mede, instead of Darius  I Hystaspis (Persian).  Please recheck, if you can ;-)

**Comment by chessurisme** [comment]
Oh okay, I overlook that. I'm sorry about this, Eric.

**Comment by erykjj** [comment]
OK, thanks to both of you!
It's done :-)

**Comment by erykjj** [comment]
Released v5.0 with all your contributions. Thank you!

**Comment by erykjj** [comment]
Hi @feriniguez 

Thank you for your suggestion. I have thought about doing that, but...
1) I do want to keep the information to a minimum and
2) the JW Library app actually makes looking up additional information very easy: select the name of the person/place, "Topics", and generally you'll find an _Insight_ book entry
